tests/test-grad0.cpp

```diff
@@ -883,9 +883,6 @@ int main(int argc, const char ** argv) {
             srand(seed);
             const int nargs = 1;
 
-            int64_t ne2[4];
-            ne2[0] = 1;
-
```

Comment:
Why was this changed?

Reply:
It's just some cleanup to remove warnings.